### PR TITLE
update orgs and units overview for clarity

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes

Restructured the orgs, units, and projects overview to more closely follow the hierarchy of these within the platform and updated some language to make it clearer what is configured at each level.

## Checklist

- [ ] The first paragraph of the page is on one line.
- [ ] The other lines have a line break at 90 characters.
- [ ] I checked the output.
- [ ] I applied the [style guide](../styleguide.md).
- [ ] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/concepts/orgs-units-projects.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+The Aiven platform uses organizations, organizational units, and projects to organize services and manage access.
+
+import Overview from "@site/static/images/content/figma/organizations-overview.png";
+
+<img alt="Hierarchy showing two organizational units, each with two projects, nested within one organization." src={Overview} width="58%" class="centered"/>
+
+## Organizations and organizational units
+
+Organizations and organizational units are collections of projects. When
```
Comment (rsalevsky) [inline docs/platform/concepts/orgs-units-projects.md:13]: I would rephrase this sentence a bit something along these lines.

An Organization enables you to run and manage Aiven services efficiently and secure. Organizational units are collection of projects and can be used to group projects together.

Maybe we can work with Janki to come up with a better wording.

File: docs/platform/concepts/orgs-units-projects.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+title: Organizations, units, and projects
+---
+
+The Aiven platform uses organizations, organizational units, and projects to organize services and manage access.
+
+import Overview from "@site/static/images/content/figma/organizations-overview.png";
+
+<img alt="Hierarchy showing two organizational units, each with two projects, nested within one organization." src={Overview} width="58%" class="centered"/>
+
+## Organizations and organizational units
```
Comment (rsalevsky) [inline docs/platform/concepts/orgs-units-projects.md:11]: I would suggest to separate these as they don't come together but one is an extension of another. We can then clearly outline what an Organization is and what an unit is.

**Review by rsalevsky — CHANGES_REQUESTED** [review]

**Review by ArthurFlag — APPROVED** [review]

File: docs/platform/concepts/orgs-units-projects.md (hunk 1 above)
Comment (staceysalamon-aiven) [reply docs/platform/concepts/orgs-units-projects.md]: I added a little of this to the intro, but let's keep it focused more on the functionality and less on the marketing-y language. :)

**Comment by staceysalamon-aiven** [comment]
Preview: https://staceys-org-docs-improvement.aiven-docs.pages.dev/docs/platform/concepts/orgs-units-projects
